client/src/components/App.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+
+import "../components/App.css";
+import {Route} from 'react-router-dom';
+import Register from './signup';
+import Signin from './signin';
+import ShowUsers from './showusers';
+
+
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <Route exact path ="/signup" component = {Register} />
```

Comment:
Somewhere in here, there should be a "/" path, so that users do not have to know they need to go to /signup or /signin.